hw3/tests/test_hw3_task01.py

```diff
@@ -0,0 +1,12 @@
+from hw3.hw3_task01 import cache_factory
+
+
+def test_with_number():
```

Comment:
What is the purpose of this test?

Reply:
To test a function that operates with numbers.  ` test_a_function_that_operates_with_numbers`

Reply:
Looks like this test is doing what you expected.   Also, you can remove `@cache_factory(times=3)` because it distracts you from testing a function that operates with numbers.

After that, you can remove the whole test, because we don't care if a function that operates with numbers is correct or not.